src/components/Calculator/ScoreTable.js

```diff
@@ -1,57 +1,112 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import Threshold from './Threshold'
 import getCourseScore from '../../utils/getCourseScore'
 import facultiesArray from '../../faculties.json'
+import thresholdsArray from '../../thresholds.json'
 
-const ScoreTable = ({ subjects }) => (
-  <div>
-    <br />
-    <br />
-    <table className='table table-bordered score-table'>
-      <tbody>
-        <tr>
-          <th scope='row'>
-            Wydział
-          </th>
-          <th scope='row'>
-            Kierunek
-          </th>
-          {/* <th scope='row'>
-            Przedmioty
-          </th> */}
-          <th scope='row'>
-            Wynik
-          </th>
-        </tr>
-        {facultiesArray.map((faculty) =>
-          faculty.courses.map((course, courseIndex) =>
-            <tr key={courseIndex}>
-              {!courseIndex &&
-              <td rowSpan={faculty.courses.length}>
-                <strong>
-                  {faculty.name}
-                </strong>
-              </td>}
-              <td>
-                {course.name}
-              </td>
-              {/* <td>
-              {course.subjects.join(', ')}
-            </td> */}
-              <td>
-                <strong>
-                  {getCourseScore(course, faculty, subjects)}
-                </strong>
-              </td>
-            </tr>
+const setupFacultiesArray = (subjects) => {
+  let i = 0
+
+  facultiesArray.forEach((faculty) => {
+    faculty.coursesScore = []
+    faculty.thresholds = []
+    faculty.courses.forEach((course) => {
+      faculty.coursesScore.push(getCourseScore(course, faculty, subjects))
+      faculty.thresholds.push(thresholdsArray[i++])
+    })
+  })
+}
+
+const ScoreTable = ({ subjects }) => {
+  setupFacultiesArray(subjects)
+
+  return (
+    <div>
+      <br />
```

Comment:
What about using margin-top on .score-table to handle spacing?
That way you would not require this `br`s and wrapping in `div`.

Also it's more flexible to use css for that kind of stuff.